src/controllers/vscodeWrapper.ts

```diff
@@ -295,6 +295,20 @@ export default class VscodeWrapper {
 		return vscode.window.showWarningMessage(Constants.extensionName + ': ' + msg);
 	}
 
+	/**
+	 * Formats and shows a vscode warning message with items
+	 */
+	public showWarningMessageAdvanced(msg: string, messageOptions: vscode.MessageOptions, items: any[]): Thenable<string> {
+		return vscode.window.showWarningMessage(Constants.extensionName + ': ' + msg, messageOptions, ...items);
```

Comment:
That's been the current design so I brought over but will address it.

Reply:
Out of curiosity - I saw you changed it but then reverted that back. What was breaking when you removed this?

Reply:
Because all our prompts include extension name, so I don't think we want to make it fall out of sync:

<img src="https://user-images.githubusercontent.com/13396919/205427316-35dc42a1-ef5c-4513-b04f-100afd5fd293.png" width="400px"/>